docs/guides/custom-schema.md

```diff
@@ -0,0 +1,223 @@
+<!--[meta]
+section: guides
+title: Custom schema
+subSection: advanced
+[meta]-->
+
+# Custom schema
+
+Out of the box, Keystone provides predictable CRUD (Create, Read, Update, Delete) GraphQL operations for [Lists](/docs/guides/schema.md#lists). They are the primary methods for updating list data and should be enough for most applications. However, custom types, queries and mutations may be added if you wish to preform non-CRUD operations.
+
+> **Hint:** See the [GraphQL philosophy](/docs/guides/graphql-philosophy.md) page for more information on how Keystone implements CRUD operations in GraphQL and when custom schema may be required.
```

Comment:
When #2654 lands, only prefixes of `'tip' | 'note' | 'important'` will get custom styles. I think I made a mistake with "hint", it feels too ambiguous and elusive.

```suggestion
> **Tip:** See the [GraphQL philosophy](/docs/guides/graphql-philosophy.md) page for more information on how Keystone implements CRUD operations in GraphQL and when custom schema may be required.
```